packages/patternfly-4/react-core/src/components/Tabs/Tabs.tsx

```diff
@@ -239,15 +237,13 @@ export class Tabs extends React.Component<TabsProps, TabsState> {
               </li>
             ))}
           </ul>
-          {!isSecondary && (
             <button
```

Comment:
The buttons should also get the modifier `pf-m-secondary` for secondary tabs

Reply:
Ok, thanks! :)